toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -60,18 +59,24 @@ class FarmServer;
 
 namespace {
 
+const char *applicationName     = "OpenToonz";
+const char *applicationVersion  = "1.1";
+const char *applicationRevision = "2";
+
 //--------------------------------------------------------------------
 TFilePath getGlobalRoot() {
   TFilePath rootDir;
 
 #ifdef _WIN32
-  TFilePath name(L"SOFTWARE\\OpenToonz\\OpenToonz\\1.0\\FARMROOT");
+  QString regpath = L"SOFTWARE\\OpenToonz\\OpenToonz\\" + applicationVersion + "\\FARMROOT";
```

Comment:
@yamisama I just figured that out, with a little google help. But thanks for replying. It builds on my machine now, hopefully it will here too, we'll see in an hour.